source/exceptions.tex

```diff
@@ -799,9 +799,7 @@
 \end{codeblock}
 The call to
 \tcode{f}
-is well-formed even though, when called,
-\tcode{f}
-might throw an exception.
+is well-formed even though it is not known whether \tcode{f} throws any exception.
```

Comment:
Since it's well-formed even if `f` definitely throws an exception (_cf._ the direct use of `throw`), we should say something like "is well-formed despite the possibility that, when called, &hellip;" or "is well-formed despite the possibility that the call to `f` throws an exception".

Reply:
How about "... despite the possibility for `f` to thrown an exception", or "... for the call to `f` to throw an exception"?

Reply:
Even better, "the call to `f` is already the subject, so just "it" suffices.